### PR TITLE
/partners page implementation

### PR DESCRIPTION
**Pull request by adamkecskes**
Asana link: https://app.asana.com/0/1193069161928913/1202001699843261/f
Screenshot:
localhost_3000_partners (8)

### CODE REVIEW

File: app/views/partners/PagePartners/index.vue [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  data() {
+    return {
+      count: 0,
+      mystring: 'hello',
+      i18nData: {
+        hyperx: `<a href='https://www.yahoo.com/now/hyperx-partners-codecombat-esports-coding-175900451.html' target='_blank'>${$.i18n.t('partners.hyperx')}</a>`,
+        respawn: `<a href='https://codecombat.com/league/#winners' target='_blank'>${$.i18n.t('partners.respawn')}</a>`,
+        forty_languages: `<a href='https://codecombat.com/contribute/diplomat' target='_blank'>${$.i18n.t('partners.forty_languages')}</a>`,
+        reach_latinx: `<a href='https://www.yahoo.com/now/mexican-superstar-eugenio-derbez-partners-200100110.html' target='_blank'>${$.i18n.t('partners.reach_latinx')}</a>`,
+        interpolation: { escapeValue: false }
```
Comment (mrfinch) [inline app/views/partners/PagePartners/index.vue:75]: ahh, u found it!! Awesome!

File: app/views/partners/PagePartners/index.vue [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+    .col-lg-12
+      a.btn.btn-primary.btn-lg(href="https://form.typeform.com/to/loZIh0I9" data-i18n="partners.talk_with_us" target="_blank")
+
+</template>
+
+<script>
+export default Vue.extend({
+  data() {
+    return {
+      count: 0,
```
Comment (mrfinch) [inline app/views/partners/PagePartners/index.vue:68]: count and mystring seem ununsed. Remove?

File: app/views/partners/PagePartners/index.vue [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+        strong(data-i18n="partners.esports_title")
+        | &nbsp;
+        span(data-i18n="[html]partners.esports_blurb" :data-i18n-options="`${JSON.stringify(i18nData)}`")
+
+  .width-container.row
+    .col-lg-12
+      p.text-navy
+        strong(data-i18n="partners.distribution_title")
+        | &nbsp;
+        span(data-i18n="[html]partners.distribution_blurb" :data-i18n-options="`${JSON.stringify(i18nData)}`")
```
Comment (mrfinch) [inline app/views/partners/PagePartners/index.vue:36]: most of them don't need all i18nOptions so could have just passed needed data directly here too. But this works too!

File: app/views/partners/PagePartners/index.vue [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+<script>
+export default Vue.extend({
+  data() {
+    return {
+      count: 0,
+      mystring: 'hello',
+      i18nData: {
+        hyperx: `<a href='https://www.yahoo.com/now/hyperx-partners-codecombat-esports-coding-175900451.html' target='_blank'>${$.i18n.t('partners.hyperx')}</a>`,
+        respawn: `<a href='https://codecombat.com/league/#winners' target='_blank'>${$.i18n.t('partners.respawn')}</a>`,
+        forty_languages: `<a href='https://codecombat.com/contribute/diplomat' target='_blank'>${$.i18n.t('partners.forty_languages')}</a>`,
```
Comment (mrfinch) [inline app/views/partners/PagePartners/index.vue:73]: We prefer relative urls so that staging.codecombat actual opens page on the same domain

**Review by mrfinch — APPROVED** [review]

**Review by mrfinch — APPROVED** [review]

**Review by nwinter — COMMENTED** [review]
Are we going to run into the same issue with Vue and `data-i18n` as [we did with /efficacy](https://github.com/codecombat/codecombat/commit/d30d89717c54ff4faf34ff2676fbb6708891cd06)?

**Comment by adamkecskes** [comment]
@mrfinch can you pls review this pr again? There's been some commits pushed since your last review.

**Review by mrfinch — APPROVED** [review]

**Pull request merged**